frontend/app/src/components/defi/loan/loans/makerdao/MakerDaoVaultDebtDetails.vue

```diff
@@ -53,5 +54,8 @@ const interest = computed(() => {
   return get(totalInterestOwed);
 });
 
+const { getIdentifierBySymbol } = useAssetInfoRetrieval();
+
+const dai: string = getIdentifierBySymbol('DAI');
```

Comment:
Do we really need to pass through the store for this? It is just a util function, right? Maybe we could use the function directly instead.